doc/saw-user-manual/analyzing-hardware-circuits-using-yosys.md

```diff
@@ -114,7 +114,18 @@ $ ghdl -a adder.vhd
 $ yosys -p 'ghdl add4; write_json adder.json'
 :::
 
-The produced file `adder.json` can then be loaded into SAW with `yosys_import`:
+> **Side Note:** this assumes the ghdl plugin was properly installed, see https://github.com/ghdl/ghdl-yosys-plugin
+> for installation details.
+
+The produced file `adder.json` can then be loaded into SAW with `yosys_import`.
+
+> **Side Note:** different versions of Yosys and GHDL may produce slightly different naming for the inner modules.
+> TabbyCAD may produce different inner module names as well, so you would need to adjust the script below
+> accordingly to account for different module names.
+>
+> The generated module names are not guaranteed to be valid Cryptol field names, so it is a good idea to sanity check
+> your JSON file before loading, and at least removing parenthesis from the names with sed:
+> `sed -i '' 's/)//g' adder.json && sed -i '' 's/(//g' adder.json`
```

Comment:
This is papering around a SAW bug, so perhaps it would be worth leaving a link to https://github.com/GaloisInc/saw-script/issues/2841 here. (That will also give us a reminder to remove this section of the tutorial once that bug is fixed.)